lib/src/components/NxTableBody/NxTableBody.tsx

```diff
@@ -45,7 +45,7 @@ const NxTableBody = function NxTableBody(props: Props) {
   return (
     <tbody {...attrs}>
       {isLoading && loadingSpinnerRow}
-      {error && !isLoading && errorRow}
+      {!!error && !isLoading && errorRow}
```

Comment:
Convert the falsey error string to an actual boolean value here. Otherwise when it's an empty string this ends up rendering whitespace in the `<tbody>` and react gives us a warning.